### DISCUSSION TITLE
`DivideByLimbCount` do not function in `StatusEffect` in `Explosion`

### DISCUSSION THREAD

**Discussion by BiffW** [FakeFishGames/Barotrauma#13291] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The affliction in the status effect will apply to every limb without being divided by limb count.
```
<Explosion range="600.0" force="50">
  <Affliction identifier="burn" strength="15" />
  <StatusEffect type="OnUse" target="Character">
    <Conditional TeamID="neq Team1" />
    <Conditional TeamID="neq FriendlyNPC" />
    <Affliction identifier="burn" strength="15" DivideByLimbCount="true" />
  </StatusEffect>
</Explosion>
```

### Reproduction steps

1. use the code provided
2. cause this explosion on enemy and see it being more powerful than expected

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.7.0 (Winter Update hotfix)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I would actually argue this is not a bug. The property only affects afflictions defined in an explosion, not afflictions caused by a StatusEffect triggered by an explosion. The StatusEffect has no "knowledge of the explosion", and won't take things like distance or obstacles between the explosion and the character into account.
